tests/test_tags.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from fastapi import APIRouter, FastAPI
+
+
+def test_string_is_invalid_in_router_tags():
+    with pytest.raises(TypeError):
+        APIRouter(tags="test")
+
+
+def test_string_is_invalid_in_router_route_tags():
+    router = APIRouter()
+
+    with pytest.raises(TypeError):
+
+        @router.get("", tags="test")
+        def test():
+            ...
+
+
+def test_string_is_invalid_in_include_router_tags():
+    app = FastAPI()
+    router = APIRouter()
+
+    @router.get("")
+    def test():
+        ...
```

Comment:
Seems this line causes code coverage to go down - it's not supposed to execute anyway. Would adding `# pragma: nocover` help?

Reply:
Another option would be to send a request and asset it returns a 404, to assure the endpoint was not created 🤔